src/executePrettier.js

```diff
@@ -37,7 +47,15 @@ const executePrettier = (editor, text) => {
           filePath: getCurrentFilePath(editor),
         }));
     }
-    return prettier.format(text, getPrettierOptions(editor));
+
+    const prettierOptions = getPrettierOptions(editor);
+    const localPrettier = getLocalPrettierPath(getCurrentFilePath(editor));
+
+    if (!localPrettier) {
+      return prettier.format(text, prettierOptions);
+    }
+
+    return getLocalPrettier(localPrettier).format(text, prettierOptions);
```

Comment:
You could move it all the way up to `main` to do it in one place and pass it through both `format` functions, I suppose? 

Although I guess we might as well hide the problem in the `helpers` by wrapping it in one function :)